MapboxVision/VisionManager/BaseVisionManager.swift

```diff
@@ -0,0 +1,197 @@
+//
+//  BaseVisionManager.swift
+//  MapboxVision
+//
+//  Created by Alexander Pristavko on 5/17/19.
+//  Copyright © 2019 Mapbox. All rights reserved.
+//
+
+import Foundation
+
+public class BaseVisionManager: VisionManagerProtocol {
```

Comment:
Can we extract `VisionManagerProtocol` extension?

Reply:
Code style issue. Will discuss in dedicated session.